CMakeLists.txt

```diff
@@ -28,7 +33,7 @@ include_directories(
 
 list(APPEND LIBS
     ${IBP_LIBRARY} ${GOP_LIBRARY} ${CZMQ_LIBRARY} ${TOOLBOX_LIBRARY}
-    ${JERASURE_LIBRARY} ${ZMQ_LIBRARY} ${FUSE_LIBRARIES}
+    ${JERASURE_LIBRARY} ${ZMQ_LIBRARY} ${FUSE_LIBRARIES} gssapi_krb5
```

Comment:
My impression is that the gssapi_krb5 dependency is specific to the gridftp sub-project so it made sense to me to pull it out of LStoreCommon.cmake since it is not "common" to all the sub-projects and put it in the list of libraries in the project specific CMakeLists.cmake. This also makes the contents of this version of LStoreCommon.cmake match the other modules.